src/spectrecoff/Spectre/Status.fs

```diff
@@ -53,3 +45,59 @@ let updateWithCustomSpinner spinner (context: StatusContext) =
         | None -> context.SpinnerStyle
 
     context
+
+let start statusText (operation: StatusOperation) =
+    task {
+        return! AnsiConsole
+            .Status()
+            .StartAsync(statusText, operation)
+    }
+
+let startWithResult<'Result> statusText (operation: StatusOperation<'Result>) =
+    task {
+        return! AnsiConsole
+            .Status()
+            .StartAsync(statusText, operation)
+    }
+
+let startWithCustomSpinner spinner (operation: StatusOperation) =
+    task {
+        let status = AnsiConsole.Status() |> configureStatus spinner
+        return! status.StartAsync(spinner.Message, operation)
+    }
+
+let startWithCustomSpinnerWithResult<'Result> spinner (operation: StatusOperation<'Result>) =
+    task {
+        let status = AnsiConsole.Status() |> configureStatus spinner
+        return! status.StartAsync(spinner.Message, operation)
+    }
+
+let startAsync statusText (operation: StatusOperationAsync) =
+    task {
```

Comment:
It makes now sense to do `task { return! ... }`, that is just unnecessary wrapping. In this case, just do 

```fs
let startAsync statusText (operation: StatusOperationAsync) =
    AnsiConsole
        .Status()
        .StartAsync(statusText, asyncToTask operation)
```

Reply:
But in general, we could (should) maybe not expose tasks at all, rather transform them to asyncs, by appending  `|> Async.AwaitTask`. Then the return type will be `Async<'T>`. What do you think?

Reply:
As discussed, we now only support `Async`